packages/core/src/interfaces/charts.ts

```diff
@@ -86,14 +86,18 @@ export interface BaseChartOptions {
 		 */
 		groupMapsTo?: string;
 		/**
-		 * used to simulate data loading
+		 * used to simulate data loading in skeleton way
 		 */
 		loading?: boolean;
 		/**
 		 * options related to pre-selected data groups
 		 * Remains empty if every legend item is active or dataset doesn't have the data groups.
 		 */
 		selectedGroups?: string[];
+		/**
+		 * used to simulate data loading in sparkline way
+		 */
+		sparklineLoading?: boolean;
```

Comment:
```suggestion
```

Reply:
Also just confirming again that there's no reference of `sparklineLoading` in the code?

Reply:
`sparklineLoading` has been removed.